test/build/core-flags/module-flags.test.js

```diff
@@ -114,6 +114,52 @@ describe("module config related flag", () => {
                             "myLoader",
                         ]);
                         expect(normalizeStdout(stdout)).toContain(`${propName}: 'pre'`);
+                    } else if (name.includes("compiler")) {
+                        let { stdout } = await run(__dirname, [`--${name}`, "test-compiler"]);
```

Comment:
Why not using `const`? There're multiple instancess in the test suite.

Reply:
We need rule in eslint config to enforce const

Reply:
yep should be const

Reply:
will add

Reply:
Let's fix and merge